data/frame.go

```diff
@@ -170,15 +170,15 @@ func (f *Field) SetConfig(conf *FieldConfig) *Field {
 	return f
 }
 
-// NewFrame returns a new instance of a Frame.
+// NewFrame returns a new Frame instance.
```

Comment:
"Instance" sounds strange to me here for some reason. I know it is copied from before, but maybe just drop it? "NewFrame returns a new Frame."

Reply:
@kylebrandt Sure, I can make the change, but it is standard language for constructing an object. You can also say "instantiates".

Reply:
I'll make the change, it doesn't really matter.

Reply:
Maybe I see what you mean, on second reading. It's just superfluous.